Filters/exclusions.txt

```diff
@@ -592,7 +592,7 @@ scorecardresearch.com
 fwmrm.net
 ! too wide
 ||ru.net^
-! replaced with *.bpmonline.com^
+! https://github.com/AdguardTeam/AdguardFilters/issues/91564
```

Comment:
wouldn't it be better to move it to the file start to keep exclusions in chronological order?

Reply:
done